src/tree/containers/ContainerProperties.ts

```diff
@@ -16,7 +17,8 @@ export const containerProperties: ITreePropertyInfo<ContainerProperty>[] = [
     { property: 'Networks', exampleValue: 'mybridge_network' },
     { property: 'Ports', exampleValue: '8080' },
     { property: 'State', exampleValue: 'exited' },
-    { property: 'Status', exampleValue: 'Exited (0) 2 hours ago' }
+    { property: 'Status', exampleValue: 'Exited (0) 2 hours ago' },
+    { property: 'Compose Project Name', description: localize('vscode-docker.tree.containers.properties.composeProjectName', 'Value used to associate containers launched by a \'compose up\' command') },
```

Comment:
```suggestion
    { property: 'Compose Project Name', description: localize('vscode-docker.tree.containers.properties.composeProjectName', 'Value used to associate containers launched by a \'docker-compose up\' command') },
```
I guess we should probably use the full name here.

Reply:
It is already a long description and user will get 'compose up'. @BigMorty ?